cli/src/lib.rs

```diff
@@ -0,0 +1,74 @@
+use std::{ffi::OsString, fs, path::PathBuf};
+
+use benchmark_common::{parse_raw, IncomingMessage, OutgoingMessage, SuiteStatus};
+use tty::{SerialConnection, SuiteConnection};
+
+pub mod tty;
+
+/// Benchmark the suite using the file provided, interpreted in raw mode.
+///
+/// Raw mode means that the files lines are parsed line by line, each representing a
+/// message that the CLI sends directly to the Suite.
+/// Implicitly sends OutgoingMessage::Done at the end.
+///
+/// For information about supported messages an their (de)serialization check the common create.
+///
+/// # Arguments
+///
+/// * `tty` - path to the tty used to communicate with the suite
+/// * `input_file` - path to the input file
```

Comment:
I am assuming input file here means for example the raw_test.bench file? Could you expand and provide a little more info on what you mean by input file?